src/main/controllers/LiveCaseStatusUpdatesController.ts

```diff
@@ -0,0 +1,15 @@
+import { Request, Response } from 'express';
+import { CourtActions } from '../resources/actions/courtActions';
+import { CourtService } from '../service/courtService';
+
+export default class LiveCaseStatusUpdatesController {
+
+  public get(req: Request, res: Response): void {
+    const courtsList = new CourtActions().getCourtsList();
```

Comment:
can this exist in the class of the line below since we dont actually need this in here?